homeassistant/components/synology_dsm/switch.py

```diff
@@ -83,6 +83,10 @@ def __init__(
         super().__init__(api, coordinator, description)
         self._version = version
 
+        self._attr_name = (
+            f"{self._api.network.hostname} Surveillance Station {description.name}"
```

Comment:
This is such a long name, does it need "Surveillance Station" added ?

Reply:
i think we should not omit "Surveillance Station" from naming, since it is used to indicate that these entities are related to the "Surveillance Station" app on the NAS - there is also already an active contribution to add the "Audio Station" app